README.md

```diff
@@ -1,83 +1,89 @@
 
 ## Configuration
 
 ### Action config
 
-The action is configured via the `custom_approvers_config.yml` file located in the `.github` subdirectory. Default config file can be overriden in workflow inputs.
-The general format is as follows.
+Action has one built-in condition check which evaluates whether PR changes any line of code containing 🔒 emoji sign or line below it.
+
+Additional condition checks can be configured via the `pr-custom-review-config.yml` file placed in the `.github` subdirectory. Default config file can be overriden in workflow step [`with`](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepswith) section. [`config_file`](#Action-config) is optional and if it is missing than only built-in check will be performed.
+
+Config file format:
 
 ```yaml
-approvals:
-  # check will fail if there is no approval
-  minimum: 1     # OPTIONAL - the same as repo protected branch settings
-  groups:        # Multiple groups with can be specified
-    - name: reviewers_group1
-      minimum: 1 # number of needed approvals from the group
-      from:
-        person: # list of individual users to request and check approvals. Works with default GITHUB_TOKEN
-          - user1
-          - user2
-    - name: reviewers_group2
-      minimum: 2
-      from:
-        person:
-          - user3
-          - user4
+approval_groups:
+  - name: CHECK NAME     # Used to create message in status check. Keep it short as description of status check has limit of 140 chars
+    condition: /^.*$/    # RegExp used to detect changes. Do not specify modifiers after closing slash. "gm" modifiers will be added
+    check_type: pr_diff  # Check type. Currently supported `pr_diff` and `pr_files`
+    min_approvals: 2     # minimum required approvals
+    users:               # GitHub users list to request review from
+      - user1
+      - user2
+    teams:               # GitHub teams list to request review from. Must be within repository organization, teams from external organizations are not supported. Specify team name(slug) only e.g 'team1' without org. 'org/team1' will lead to failure.
+      - team1
+      - team2
 ```
 
 ### Workflow config
 
-Once the `custom_approvers_config.yml` file is in place, add the action to execute on every PR and then set its status as required to start enforcing your new approval policy!
-
 ```yaml
-name: 'PR-CUSTOM-REVIEW' # Name is used for creation of the PR commit status
+name: PR Custom Review Status                     # used to create status check name
 
-on:
+on:                                               # Events which triggers action
   pull_request:
+    branches:
+      - main
+      - master
     types:
```

Comment:
Please mention the necessary types, so we don't miss anything and don't run it without need.